src/main/java/taxi/controller/car/GetAllCarsController.java

```diff
@@ -1,24 +1,24 @@
 package taxi.controller.car;
 
 import java.io.IOException;
-import java.util.List;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import taxi.lib.Injector;
-import taxi.model.Car;
 import taxi.service.CarService;
 
+@WebServlet(urlPatterns = "/cars")
 public class GetAllCarsController extends HttpServlet {
     private static final Injector injector = Injector.getInstance("taxi");
-    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
 
     @Override
-    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Car> cars = carService.getAll();
-        req.setAttribute("cars", cars);
-        req.getRequestDispatcher("/WEB-INF/views/cars/all.jsp").forward(req, resp);
+        req.setAttribute("cars", carService.getAll());
+        req.getRequestDispatcher("WEB-INF/views/cars/cars.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/cars.jsp").forward(req, resp);
```
add `/` in all your controllers before WEB-INF